### PR TITLE
Added: Checkbox

### PR DESCRIPTION
**Pull request by thesanjeevsharma**
Review and Merge

### CODE REVIEW

File: src/assets/icons/Tick.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Icon = () => {
```
Comment (prvnbist) [inline src/assets/icons/Tick.jsx]: Instead of just `Icon` use the full name `TickIcon`

File: src/assets/icons/index.js [hunk 2]
```diff
@@ -4,12 +4,14 @@ import ArrowDownIcon from './ArrowDownIcon'
 import ArrowUpIcon from './ArrowUpIcon'
 import RemoveIcon from './RemoveIcon'
 import PlusIcon from './PlusIcon'
+import Tick from './Tick'
```
Comment (prvnbist) [inline src/assets/icons/index.js]: Similarly, when it comes to icons, use same naming convention - `TickIcon` so we differentiate it from a component

**Comment by prvnbist** [comment]
The label for the checkbox breaks to new line, shouldn't it be on same line as the checkbox?

**Comment by thesanjeevsharma** [comment]
I've used a span for Checkbox and Label, both. This shouldn't have happened; I'll see what's the issue.

File: src/assets/icons/index.js (hunk 2 above)
Comment (thesanjeevsharma) [reply src/assets/icons/index.js]: Renamed!

File: src/assets/icons/Tick.jsx (hunk 1 above)
Comment (thesanjeevsharma) [reply src/assets/icons/Tick.jsx]: Well, in the file it doesn't matter as it's a default export. But anyway, I'll rename this.

Comment (prvnbist) [reply src/assets/icons/Tick.jsx]: Yea, inside file it doesn't matter.

**Pull request merged**